libraries/alicloud_backend.rb

```diff
@@ -34,9 +34,9 @@ def alicloud_client(api:, api_version:)
                    "https://#{api}.aliyuncs.com"
                  else
                    # AliCloud VPN endpoints vary between regions, so the following accounts for that variability
-                   if api == "vpn" && %w{cn-qingdao cn-beijing cn-beijing cn-shanghai cn-shenzhen cn-hongkong ap-southeast-1 us-west-1 us-east-1 cn-shanghai-finance-1 cn-shenzhen-finance-1 cn-north-2-gov-1}.include(region)
+                   if %w{vpc slb}.include(api) && %w{cn-qingdao cn-beijing cn-beijing cn-shanghai cn-shenzhen cn-hongkong ap-southeast-1 us-west-1 us-east-1 cn-shanghai-finance-1 cn-shenzhen-finance-1 cn-north-2-gov-1}.include(region)
                      "https://#{api}.aliyuncs.com"
-                   else
+                   elsif api == "ecs" && %w{cn-hongkong ap-southeast-1 us-west-1 us-east-1 cn-north-2-gov-1}.include(region)
```

Comment:
@jonlives as we discussed last time, this is a not a scalable approach and we are pretty soon going to end up having all the api type handled here with a very big conditional block..

Reply:
Agreed, and this is definitely a temporary fix - I've verified with the customer that this current list represents the full list of services they're currently using so am not expecting further changes in the near term.

Reply:
Okay, considering this as a short term fix

Reply:
there is a typo, it should be `include?(region)` and not `include(region)`